default/stringtables/en.txt

```diff
@@ -1515,6 +1515,9 @@ Limit of client message size in bytes.
 OPTIONS_DB_DROP_EMPIRE_READY
 Drop empire's readiness on joining to the playing game.
 
+OPTIONS_DB_TAKE_OVER_AI
+Server option to allow human player to delegate any AI player.
```

Comment:
I would interpret this as saying that the human player can say that any AI player can replace the player.

I think you mean that this option would allow any human player to replace any AI player?

Reply:
Yes, that what I mean.

Reply:
Is a player with no assigned empire automatically assigned an AI empire if there is one or multiple AI empires, or do they get put into the lobby in order to pick one?

Reply:
Player without assigned empire will automatically be assigned to a AI empire if there is only one AI empire. This behaviour could be changed.

Reply:
What if there are multiple AI empires?

Reply:
Player will get to lobby to choose.

Reply:
OK, then I guess
````
OPTIONS_DB_TAKE_OVER_AI
Server option to allow human players to take control of AI empires.
````

Reply:
Changed.